inc/compatibilities/sassy_social_share.php

```diff
@@ -0,0 +1,63 @@
+<?php
+
+/**
+ * Class Optml_sassy_social_share.
+ *
+ * @reason Sassy social share picks eco images to share
+ */
+class Optml_sassy_social_share extends Optml_compatibility {
+	/**
+	 * Should we load the integration logic.
+	 *
+	 * @return bool Should we load.
+	 */
+	function should_load() {
+		include_once( ABSPATH . 'wp-admin/includes/plugin.php' );
+
+		if ( is_plugin_active( 'sassy-social-share/sassy-social-share.php' ) ) {
+			$ss_options = get_option( 'heateor_sss' );
+			$ss_bars = ['vertical_re_providers', 'horizontal_re_providers'];
+			foreach ( $ss_bars as $key => $bar ) {
+				foreach ( $ss_options[ $bar ] as $index => $value ) {
+					if ( isset( $value ) && is_string( $value ) ) {
+						if ( strpos( $value, 'pinterest' ) !== false ) {
+							return true;
+						}
+					}
+				}
+			}
+		}
+		return false;
+	}
+
+	/**
+	 * Register integration details.
+	 */
+	public function register() {
+		add_action(
+			'wp_head',
+			function () {
+				$output = '
+		<script type="application/javascript">
+			(function(w, d){ 
+					w.addEventListener("load", function(){
+						let p=d.querySelectorAll( ".heateorSssSharing.heateorSssPinterestBackground" );
```

Comment:
We can use wp_add_inline_script.

Reply:
using this will require to add an empty script above what we add

Reply:
no, we can use false for the handle